iOS/DuckDuckGo/DuckPlayer/NativeUI/Views/DuckPlayerContainer.swift

```diff
@@ -122,55 +141,123 @@ private struct SheetView<Content: View>: View {
+
+                        Rectangle()
+                            .fill(Color.clear)
+                            .frame(height: DuckPlayerContainer.Constants.dragAreaHeight)
+                            .contentShape(Rectangle())
+                            .gesture(
+                                DragGesture()
+                                    .updating($dragStartOffset) { _, state, _ in
+                                        if state == nil {
+                                            state = sheetOffset
+                                            viewModel.setDragging(true)
+                                        }
+                                    }
+                                    .onChanged { value in
+                                        guard let dragStartOffset else { return }
+
+                                        let offsetY = value.translation.height
+                                        if offsetY > 0 {
+                                            withAnimation(.spring(duration: 0.3, bounce: 0.2)) {
+                                                sheetOffset = dragStartOffset + offsetY
+                                            }
+                                        } else if offsetY < 0 {
+                                            let y = 1.0 / (1.0 + exp(-1 * (abs(offsetY) / 50.0))) - 0.5
+                                            withAnimation(.spring(duration: 0.3, bounce: 0.2)) {
+                                                sheetOffset = dragStartOffset + y * max(offsetY, -50)
+                                            }
+                                        }
+                                    }
+                                    .onEnded { value in
+                                        viewModel.setDragging(false)
+                                        let offsetY = value.translation.height
+
+                                        if offsetY > DuckPlayerContainer.Constants.dragThreshold || value.velocity.height > 50 {
+                                            onDismiss()
+                                        } else if offsetY < -DuckPlayerContainer.Constants.dragThreshold || value.velocity.height < -50 {
+                                            isAnimatingToTop = true
+
+                                            // Start presenting DuckPlayer immediately
+                                            onPresentDuckPlayer()
+
+                                            // Animate the pill to top and fade out
+                                            withAnimation(.easeOut(duration: 0.3)) {
+                                                opacity = 0
+                                            }
+                                            withAnimation(.spring(duration: 0.5, bounce: 0.2)) {
+                                                sheetOffset = -containerHeight
```

Comment:
Can this be combined into one? 

```
withAnimation(.spring(duration: 0.5, bounce: 0.2)) {
    sheetOffset = -containerHeight
    opacity = 0
}
```

Reply:
I'll just keep one.  it's barely visible.